app/questionPage.html

```diff
@@ -35,24 +35,36 @@
     <script src="js/dist/sendResults.js" defer></script>
     <script src="js/dist/overview.js" defer></script>
     <script src="https://kit.fontawesome.com/2373a66060.js" defer></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.0.3/jquery.min.js"></script>
+    <script src="//netdna.bootstrapcdn.com/bootstrap/3.0.2/js/bootstrap.min.js"></script>
 </head>
 <body>
-<!--question number and time -->
+<!--instructions button and logo -->
 <div id="question_page">
     <div class="container">
         <div class="row">
-            <button type="button" id="instructions" class="instructions btn btn-info left">Instructions</button>
-            <img class="logo" alt="iO Academy Logo" src="./images/logos/SVGs/iO_Logo_Alt.svg">
-        </div>
-        <div class="row">
-            <div class="left">
-                <h4>Question: </h4>
+            <div class="col-6">
+                <button type="button" class="btn btn-info left instructions-button" data-toggle="modal" data-target="#instructionsModal">Instructions</button>
             </div>
-            <div class="left">
-                <h4 id="question-counter"></h4>
+            <div class="col-6">
+                <img class="logo" alt="iO Academy Logo" src="./images/logos/SVGs/iO_Logo_Alt.svg">
             </div>
-            <div class="col-md-2 right">
-                <h4 class="timer"></h4>
+        </div>
+        <!--instructions modal -->
+        <div class="modal fade" id="instructionsModal" tabindex="-1" role="dialog" aria-labelledby="purchaseLabel" aria-hidden="true">
+            <div class="modal-dialog">
+                <div class="modal-content">
+                    <div class="modal-header">
+                        <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
```

Comment:
Did you manage to get the data dismiss/data toggle working in the end. If so, great. If not, they can _probably_ be removed to avoid confusion for future cohorts.